lib/meilisearch-rails.rb

```diff
@@ -843,7 +843,7 @@ def meilisearch_settings_changed?(server_state, user_configuration)
           if user.is_a?(Hash) && server.is_a?(Hash)
             meilisearch_settings_changed?(server, user)
           elsif user.is_a?(Array) && server.is_a?(Array)
-            user.map(&:to_s) != server.map(&:to_s)
+            user.map(&:to_s).sort! != server.map(&:to_s).sort!
```

Comment:
Why did you have to use `sort!` and not just `sort` here?

Reply:
Mainly out of frustration. While I understand objectively that this function is very short and does not deal with a lot of data, the type conversions really bother me and it feels like it performs terribly.

So I looked up performance comparisons for different methods to compare arrays regardless of order and `sort!` was slightly faster than `sort`.

Of course these arrays are tiny and the difference does not matter one bit, especially since `map` already crates new arrays.